### PR TITLE
refactor: upgrade mdn-data

### PR DESCRIPTION
**Pull request by TrySound**
Ref https://github.com/webstudio-is/webstudio/issues/2647

Here upgraded to the latest mdn data and improved properties detection. Excluded experimental and undocumented css properties.

- font-synthesis got longhands in mdn data
- background-position-x/y got excluded as experimental (though support is good) but we anyway write as shorthand
- excluded white-space-collapse, text-wrap and text-wrap-* until decide how to maintain them in the future
- field-sizing and zoom are new additions

### CODE REVIEW

**Review by kof — APPROVED** [review]

File: packages/css-data/src/custom-data.ts [hunk 1]
```diff
@@ -18,9 +18,16 @@ export const valueTypes = [
...
+  "dashed-ident",
   "single-animation-composition",
   "palette-identifier",
   "flex",
+  "inset-area",
+  "offset-path",
+  "coord-box",
+  "anchor-element",
+  "try-tactic",
+  "try-size",
```
Comment (TrySound) [inline packages/css-data/src/custom-data.ts:30]: I'm not sure if there is any use in these types

Comment (kof) [reply packages/css-data/src/custom-data.ts:30]: I think the question is rather: would it hurt to support as much as possible?

Comment (kof) [reply packages/css-data/src/custom-data.ts:30]: We are saying we support full css on the site

Comment (TrySound) [reply packages/css-data/src/custom-data.ts:30]: Those are not properties but css types

Comment (kof) [reply packages/css-data/src/custom-data.ts:30]: values, properties, all part of css, right?

Comment (TrySound) [reply packages/css-data/src/custom-data.ts:30]: I mean specifically this list is used only to detect custom idents

**Pull request merged**